internal/timewarp/timewarp_test.go

```diff
@@ -60,6 +60,11 @@ func TestHandler_ServeHTTP(t *testing.T) {
 						},
 					},
 				},
+				URLValidator: func(expected, actual string) {
```

Comment:
Could you not extract this out somwhere as a utility?

Reply:
definitely could do so. it's been an easy copy-paste but it's being used across more of our tests. i can add this in a follow-on.